sql/core/src/test/resources/sql-tests/results/show-tables.sql.out

```diff
@@ -128,62 +128,108 @@ SHOW TABLE EXTENDED
 
 
 -- !query 13
-SHOW TABLE EXTENDED LIKE 'show_t1' PARTITION(c='Us')
+SHOW TABLE EXTENDED LIKE 'show_t1' PARTITION(c='Us', d=1)
 -- !query 13 schema
-struct<>
+struct<database:string,tableName:string,isTemporary:boolean,information:string>
 -- !query 13 output
-org.apache.spark.sql.catalyst.parser.ParseException
-
-Operation not allowed: SHOW TABLE EXTENDED ... PARTITION(line 1, pos 0)
-
-== SQL ==
-SHOW TABLE EXTENDED LIKE 'show_t1' PARTITION(c='Us')
-^^^
+show_t1
```

Comment:
`hiveResultString` is only used for testing. I think we should fix it as long as it does not break any test case.